db/migrate/20230222155905_add_author_and_post_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorAndPostToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :author, foreign_key: { to_table: 'users' }
+    add_reference :comments, :post, foreign_key: { to_table: 'posts' }
+  end
+end
```

Comment:
- [ ] This table is not required, please 🙏🏽 remove it. We just need user, post, comments, and likes tables.